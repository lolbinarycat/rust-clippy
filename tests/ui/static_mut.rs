#![warn(clippy::static_mut)]

static mut NUMBER: u8 = 3;
