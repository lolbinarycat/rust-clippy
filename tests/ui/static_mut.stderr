error: declaration of static mut
  --> tests/ui/static_mut.rs:3:1
   |
LL | static mut NUMBER: u8 = 3;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: remove the `mut` and use a type with interior mutibability that implements `Sync`, such as `std::sync::Mutex`
   = note: `-D clippy::static-mut` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::static_mut)]`

error: aborting due to 1 previous error

